canonical-web-and-design/ubuntu.com
# Clean-up redirects

## Done

- Updated a several redirects
    - shop.u.c is dead
    - index_kylin should just go to cn.ubunut.com
    - community.u.c is now discourse.u.c
    - jujucharms.com is now jaas.ai
    - etc...
- Add shop.u.c to deleted

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Test some redirects
- See that the list looks reasonable
